### PR TITLE
Fix temperature default value and helper text

### PR DESCRIPTION
**Pull request by DonggeLiu**
Be more specific about the range for different models:
1. `[0-2]` for `1.5` models
2. `[0-1]` for `1` models.

### CODE REVIEW

**Comment by DonggeLiu** [comment]
The [context window size](https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models#:~:text=1%20million%20tokens.-,Max%20total%20tokens%20(input%20and%20output)%3A%201%C2%A0MB%0AMax%20input%20tokens%3A%202%C2%A0MB,-Max%20output%20tokens) is a bit confusing (part of long prompts might not be used), but I reckon 2M is fine for now.

File: experimental/manual/prompter.py [hunk 1]
```diff
@@ -32,8 +32,9 @@ def parse_args() -> argparse.Namespace:
       '--temperature',
       type=float,
       default=TEMPERATURE,
-      help=('A value between 0 and 1 representing the variety of the targets '
-            'generated by LLM.'))
+      help=('A valuere presenting the variety of the targets generated by '
```
Comment (inferno-chromium) [inline experimental/manual/prompter.py:35]: typo value

**Review by inferno-chromium — APPROVED** [review]

File: experimental/manual/prompter.py (hunk 1 above)
Comment (DonggeLiu) [reply experimental/manual/prompter.py]: Thanks fixed!

**Pull request merged**